### DISCUSSION TITLE
Integrare Microsoft Outlook in HA

### DISCUSSION THREAD

**Discussion by Borghese** [indomus/forum#2903] [GENERALE - Discussioni generiche]
Ciao a tutti,
premetto che ho fatto una rapida ricerca su web ma non ho trovato nulla di specifico.
Mi chiedevo se fosse possibile integrare in qualche modo il client di posta Microsoft Outlook in HA

La richiesta nasce dal fatto che avrei necessità di una macchina che sia up&running 24/7 per l'esecuzione di alcuni script VBA scritti in outlook appunto, da eseguire alla ricezione di alcuni tipi di mail
Volevo capire se riuscissi a sfruttare il fatto di avere il mini pc che ospita HA sempre acceso, e installare tutto qua dentro quindi

Sapete aiutarmi ?

**Comment by 50494554524F** [comment]
da outlook puoi chiamare dei link webhook di ha e scambiare dati... da ha puoi solo usare le integrazioni shell_command e command_line

**Reply by Borghese** [reply]
Sono riuscito a installare e configurare da HACS l'integrazione Office365 seguendo questa guida : https://rogerselwyn.github.io/O365-HomeAssistant/calendar_panel.html
Ora in HA ho delle entità che rilevano, tra l'altro, la presenza di mail provenienti da un determinato indirizzo (settato in config.yaml)

Ora, come posso configurare su HA l'inoltro di queste mail ? è questa la parte che non mi è chiara...

**Reply by 50494554524F** [reply]
te l'ho scritto, shell_command e command_line, e chiami quello che devi chiamare... se lo vuoi già fatto non lo so

**Reply by Borghese** [reply]
ah ok scusa non avevo capito... ok ora cerco info sul web per capire.... grazie intanto

**Reply by 50494554524F** [reply]
ps: eventualmente puoi vedere se funziona bene inglobare la web app con 

https://www.home-assistant.io/dashboards/iframe/

**Reply by Borghese** [reply]
purtroppo la web app non mi aiuta. Lo script WBA che deve girare è solo in locale, quindi da web non riuscirei a richiamarlo. Ma è complicato, sto pensando di optare per soluzione di virtualizzazione windows su una macchina dedicata (raspi o minipc) e ci installo direttamente il pacchetto office...
